Sources/SwiftFCSDKiOS/SwiftFCSDKiOS.docc/Version-4.2.0.md

```diff
@@ -0,0 +1,83 @@
+```swift
+@available(iOS 15, *)
+@MainActor @objc final public func previewBufferView() async -> UIView?
+```
+
+```swift
+@available(iOS 15, *)
+@objc final public func removePreviewView() async
+```
+
+### Sample Buffer View
+This method is designed to create a metal rendered `UIView` for your remote video during a video call. This method must be used if you desire to use our **Picture in Picture** feature.
+When a call is finished the **SDK** consumer is required to clean up the call by calling the `removeBufferView` method. 
+```swift
+@available(iOS 15, *)
+@MainActor @objc final public func sampleBufferView() async -> UIView?
+```
+
+```swift
+@available(iOS 15, *)
+@objc final public func removeSampleView() async
+```
+
+### AVCaptureSession
+In order to check if your devices support Multitasking Camera Access, when a **PreviewBufferView** is created we can get the **AVCaptureSession** from the **PreviewBufferView**. If your Device is supported by Apple, you can use the new **AVPictureInPictureVideoCallViewController** provided by Apple. Otherwise you will need to continue using **AVPictureInPictureController** and request permission from Apple for the proper entitlement. This method only works if you are using **PreviewBufferView** for your Local Video UIView.
+```swift
+@available(iOS 15, *)
+@objc final public func captureSession() async -> AVCaptureSession?
+```
+
+An example in checking for the method is as follows:
+```swift
+if captureSession.isMultitaskingCameraAccessSupported {
+// Use AVPictureInPictureVideoCallViewController
+} else {
+// Use AVPictureInPictureController
+}
+```
+
+### Set PiP Controller
+Each **ACBClientCall** object has a `setPipController` method that is required for Picture in Picture. This method informs the **SDK** who the **AVPictureInPictureController** is and allows us to receive its delegate methods. This is required for us to handle the **SampleBufferView** accordingly when tha app activates Picture in Picture.
+```swift
+@available(iOS 15, *)
+@MainActor @objc final public func setPipController(_ controller: AVPictureInPictureController) async
+```
+
+### Feed Background Image
+In order to use the **Virtual Background** feature the **SDK** consumer is required to feed the **SDK** a UIImage that the **SDK** can use as the virtual background. It is **STRONGLY** encouraged for performance reasons to feed an image that is as small as possible. This will save your application from excessive memory and cpu usage. If you fail to feed the SDK a smaller **UIImage** we will automatically adjust the size to an acceptable limit. If you wish to blur the background instead of using an image, you can select **.blur** from the **VirtualBackgroundMode** enum. You are still required to feed an image. Virtual Backgrounds can also be blurred. If you have set a virtual background and only want to blur your real background, you must first remove the virtual background using the method `removeBackgroundImage()`. 
```

Comment:
"If you fail to feed the SDK a smaller **UIImage** we will automatically..." replace with "If you fail to feed the SDK a small **UIImage**, the SDK will automatically...

Why would it be required for someone desiring blurred background to provide an image? If so, that would seem to be a code error. Surely the documentation is wrong here, especially as it goes on to say that if you want to blur the real background, you have to remove the virtual background image?